src/main/java/com/binance/dex/api/client/crosschain/UnsignedNumber.java

```diff
@@ -0,0 +1,34 @@
+package com.binance.dex.api.client.crosschain;
+
+import com.binance.dex.api.client.encoding.serializer.UnsignedNumberSerializer;
+import com.binance.dex.api.client.rlp.RlpDecodable;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import java.math.BigInteger;
+
+@JsonSerialize(using = UnsignedNumberSerializer.class)
+public class UnsignedNumber implements RlpDecodable {
+
```

Comment:
This class is designed to decode byte array to unsigned number, since the  number represented by the RLP encoded bytes  is always be unsigned while there is no unsigned number type defined in Java.